apiserver/JobSubmission.md

```diff
@@ -13,7 +13,7 @@ The first step is to deploy KubeRay operator and API server.
 Reffer to [readme](README.md) for setting up KubRay operator and API server.
```

Comment:
```suggestion
Refer to [readme](README.md) for setting up KubeRay operator and API server.
```
Thanks for the followup! I think the doc comments in your prior PR were not applied or overwritten by a rebase, for example this line. It would be great if you could make a followup PR for those too at some point.